webpack.rules.ts

```diff
@@ -32,4 +32,11 @@ export const rules: Required< ModuleOptions >[ 'rules' ] = [
 			},
 		},
 	},
+	/**
+	 * PHP WASM includes binary files from the intl library in the web/shared directory.
+	 */
+	{
+		test: /node_modules\/@php-wasm\/web\/shared\/.*\.dat$/,
+		use: 'ignore-loader',
+	},
```

Comment:
Let's remove the `ignore-loader` dependency. I think we can use directly:

```suggestion
	{
		test: /\.dat$/,
		use: 'asset/resource',
	},
```

That also fixed the make on my Windows.

Reply:
Thanks! This fixed the issue for me locally.